src/controllers/ContactController.php

```diff
@@ -321,17 +321,14 @@ public function actionUpdateEmployee($id)
         }
 
         $model = new EmployeeForm($contact, 'update');
-        $validationResult = $model->validate();
-        if (Yii::$app->request->isPost && $validationResult === true && $model->load(Yii::$app->request->post())) {
-            $saveResult = $model->save();
-            if ($saveResult === true) {
+        if (Yii::$app->request->isPost) {
+            if ($model->load(Yii::$app->request->post()) && $model->validate() && $model->save()) {
                 Yii::$app->session->addFlash('success', Yii::t('hipanel:client', 'Employee contact was save successfully'));
```

Comment:
was saveD
```suggestion
                Yii::$app->session->addFlash('success', Yii::t('hipanel:client', 'Employee contact was saved successfully'));
```
и надо поправить перевод